src/LegacyPortal.js

```diff
@@ -76,11 +76,12 @@ class Portal extends React.Component {
     // Save reference for future access.
     if (overlay !== null) {
       this._mountOverlayTarget();
-      ReactDOM.unstable_renderSubtreeIntoContainer(
-        this, overlay, this._overlayTarget, (inst) => {
-          this._overlayInstance = inst;
```

Comment:
turns out this is the react component instance... since the immediately rendered thing for the modal is a `<div>`, it ends up being `undefined`